kusto/ingest/streaming.go

```diff
@@ -15,7 +17,7 @@ import (
 
 type streamIngestor interface {
```

Comment:
streamConn

Reply:
Why? it's a good name for what the interface does. I hate the "conn" terminology.